pkg/instrumentation/tracing/tracing.go

```diff
@@ -103,28 +104,33 @@ func Start(options ...Option) error {
 
 // Stop tracing.
 func Stop() {
-	trc.shutdown()
+	trc.flush()
```

Comment:
Why not call `trc.shutdown()`, as there is such a function. If calling flush is better, then perhaps consider removing `shutdown()` as it does not seem to be called anywhere.

Reply:
Because I was supposed to remove `shutdown()` altogether, but I forgot. Thanks for spotting it, updated.